pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml-elements
+
+> Extract elements and structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements of an XML document (as XPATH expressions):
+
+`xml elements {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements and attributes of an XML document:
+
+`xml elements -a {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements, attributes, and their values:
+
+`xml elements -v {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Print sorted unique elements of an XML document (to see structure):
```

Comment:
Perhaps this will read better?
```suggestion
- Print sorted unique elements from an XML document (to see structure):
```

Reply:
xml-elements.md:- Print sorted unique elements from an XML document to see its structure:
xml-elements.md:- Print sorted unique elements from an XML document up to a depth of 3: